### PR TITLE
replicate the bug fixes #92

### LINKED ISSUE #92: get_initial_value bug and possible get_equations_for bug
I think I have discovered a bug (let me know if I'm missing something!) 
This bug is preventing me from continuing with chaste code gen!
The model which I'm working with is attached

What I'm doing is:
`membrane_capacitance = self._model.get_symbol_by_ontology_term(self._OXMETA, "membrane_capacitance")`
`initial_value = self._model.get_initial_value(membrane_capacitance))`
and I get a `KeyError`

The model has a membrane_capacitance with an initial value.

Second issue, but I am less sure of this: should the capacitence not show up in the following? (it doesn't):
`membrane_stimulus_current = self._model.get_symbol_by_ontology_term(self._OXMETA, "membrane_stimulus_current")`
`self._model.get_equations_for([membrane_stimulus_current ])`

[aslanidi_model_2009.zip](https://github.com/ModellingWebLab/cellmlmanip/files/3676516/aslanidi_model_2009.zip)

### PR DESCRIPTION
**Pull request by skeating**
Looking into bug https://github.com/ModellingWebLab/cellmlmanip/issues/92
Fixes #92

### CODE REVIEW

File: cellmlmanip/model.py [hunk 1]
```diff
@@ -643,7 +643,13 @@ def get_initial_value(self, symbol):
         :param symbol: Sympy Dummy object of required symbol
         :return: float of initial value
         """
-        return float(self.graph.nodes[symbol]['initial_value'])
+        # fixed for issue where graph.nodes[symbol] does not have an 'initial_value' entry
```
Comment (jonc125) [inline cellmlmanip/model.py]: s/fixed/Fix/

File: cellmlmanip/model.py [hunk 2]
```diff
@@ -643,7 +643,13 @@ def get_initial_value(self, symbol):
...
+        node = self.graph.nodes[symbol]
+        if 'initial_value' in node:
+            return float(node['initial_value'])
+        # no initial_value entry try the dummy_metadata
```
Comment (jonc125) [inline cellmlmanip/model.py]: Would it be better just to do this approach always? Then we wouldn't need any comments here, just the two lines of code below!

File: tests/test_aslanidi_model.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            os.path.dirname(__file__),
+            "cellml_files",
+            "aslanidi_model_2009.cellml"
+        )
+        return load_model(cellml)
+
+    def test_initial_value_capacitance(self, model):
+        model.get_equation_graph()
+        membrane_capacitance = model.get_symbol_by_ontology_term(OXMETA, "membrane_capacitance")
+        # was raising KeyError but should not
```
Comment (jonc125) [inline tests/test_aslanidi_model.py]: This commented out bit should be removed.

File: tests/test_unit_conversion.py [hunk 4]
```diff
@@ -89,4 +89,3 @@ def test_conversion_factor_same_units(simple_model):
     assert simple_model.units.get_conversion_factor(to_unit=to_unit, from_unit=from_unit) == 1
     # expression to unit
     assert simple_model.units.get_conversion_factor(to_unit=to_unit, expression=expression) == 1
-
```
Comment (jonc125) [inline tests/test_unit_conversion.py]: Isn't it best practice to keep a newline at the end of source files?

File: cellmlmanip/model.py (hunk 2 above)
Comment (skeating) [reply cellmlmanip/model.py]: My rationale was that the original seemed to work for most things but I've simplified it until we hit a case where that one doesnt work :-)

**Comment by skeating** [comment]
@MauriceHendrix since you reported the bug it would be great if you check this fixes your issue

**Comment by MauriceHendrix** [comment]
> @MauriceHendrix since you reported the bug it would be great if you check this fixes your issue

Works great thanks!

**Pull request merged**